main.go

```diff
@@ -179,7 +179,11 @@ func visit(url *url.URL) {
 		fmt.Println(grayscale(14)(k+":"), color.CyanString(strings.Join(resp.Header[k], ",")))
 	}
 
-	fmt.Println("\nBody discarded\n")
+	if !followRedirects {
```

Comment:
You always want to print the body even if following redirects.

However the code that prints the body or writes it to a file should be aware that certain status codes and http methods do not include a body in the response. For example HEAD / HTTP/1.1 will return headers, but no body, or 302 Moved should not return a body (according to the spec)

This should probably be handled later in the method